enum/Analytics.ts

```diff
@@ -55,6 +75,60 @@ export const Events: Record<string, IAnalyticsEvent> = {
 			sf_chat_key: '',
 		},
 	},
+	[EventName.ESCALATION_FAILED_DUE_TO_ERROR_GETTING_SF_MESSAGES]: {
+		category: EventCatagory.ESCALATION,
+		eventType: EventType.SESSION,
+		action: 'failed',
+		properties: {},
+	},
```

Comment:
I think we missed faliure_reason here. Or Is this even an escalation error? we are calling it in checkCurrentChatStatus.

Reply:
And SubscribeToLiveAgentHelper. Sends even when we are not able to pull messages for SF. Do we want an event for that?

Reply:
If it fails the escalation or closes the chat then yes